shopping_mall/backend/app/routers/chatbot.py

```
@@ -477,3 +480,53 @@ def get_session_messages(
+    from app.models.tool_metric import ToolMetric
+
+    days = _PERIOD_DAYS.get(period)
+    if not days:
+        raise HTTPException(status_code=400, detail=f"지원하지 않는 기간: {period}. 사용 가능: 1d, 7d, 30d")
+
+    cutoff = datetime.now(timezone.utc) - timedelta(days=days)
+
+    query = db.query(
+        ToolMetric.tool_name,
+        func.count().label("call_count"),
+        func.avg(func.cast(ToolMetric.success, Integer)).label("success_rate"),
```

Ruff (0.15.10)

[error] Undefined name `Integer`

(F821)